### ISSUE TITLE
Doubts with function adjust_predicts

### ISSUE THREAD

**Issue by AitorDomec** [imperial-qore/TranAD#50]
Inside the code, in line 340 on the _main.py_ you get the results calling the _pot_eval_ function defined in the _pot.py_ file. Inside _pot_eval_ you call the _adjust_predicts_ function defined in lines 28-74 of _pot.py_. That function is adjusting the predicted labels based on the real ones, that does not seem correct to me.
¿Is it really correct to work like that?
Maybe I am missing something.

**Comment by shreshthtuli** [comment]
As is common in prior work ([MTAD-GAT](https://github.com/ML4ITS/mtad-gat-pytorch/blob/main/eval_methods.py) and [OmniAnomaly](https://github.com/NetManAIOps/OmniAnomaly/blob/master/omni_anomaly/eval_methods.py)), we this hyperparameter threshold to adjust predictions and generate anomaly labels.

**Comment by AitorDomec** [comment]
Is it really correct to adjust the supposedly detected anomalies based on the real labels? Is this a commonly accepted method for anomaly detection?
I think that sharing your work with the world is very advantegeous and should be done more, so I thank you for it.

**Comment by shreshthtuli** [comment]
This is a common method used in prior work and we use the same approach for fair comparison. We are glad you found this codebase helpful.
